gammapy/datasets/fermi.py

```diff
@@ -215,6 +217,158 @@ def fetch_fermi_diffuse_background_model(filename='gll_iem_v02.fit'):
 
     return filename
 
+class Fermi3FGLObject(object):
+    """
+    Class representing an object in the Fermi 3FGL catalog.
+    """
+
+    from astropy.units import Quantity
+    from ..spectrum import EnergyBounds
+
+    # Fermi catalog is lazily loaded on first access
+    # and cached at class level (not instance level)
+    fermi_cat = None
```

Comment:
Same here ... I'd say make this private, i.e. rename the attribute to `_fermi_cat`.